graphql_compiler/compiler/expressions.py

```diff
@@ -545,6 +592,16 @@ def to_cypher(self):
 
         return template.format(mark_name=mark_name, field_name=field_name)
 
+    def to_sql(self, aliases, current_alias):
+        """Return a sqlalchemy Column picked from the appropriate alias."""
+        if isinstance(self.field_type, GraphQLList):
+            raise NotImplementedError(u'The SQL backend does not support lists.')
+
+        if '@' in self.location.field:
+            raise NotImplementedError(u'The sql backend does not support typename.')
+
+        return aliases[self.location.at_vertex().query_path].c[self.location.field]
```

Comment:
Should we maybe have a helper function that takes `aliases` and a location object and returns `aliases[self.location.at_vertex().query_path].c[self.location.field]` after doing all the checks for unsupported cases? Seems like this code exists in a few places.

Reply:
It appears twice. If I create a helper, all it would do is force the reader to hop up and down to understand the code (ruins locality in reading). Do you see a problem that the change would solve?